events/elections/2021/README.md

```diff
@@ -6,6 +6,17 @@ The role of this election is to fill out the four (4) seats due for
 reelection this year on the [Kubernetes Steering Committee]. Each elected
 member will serve a two (2) year term.
 
+## Changes
+
+This year is the first year we will be using [Elekto] to conduct the election.
+Elekto is a new application, commissioned as an internship by the CNCF.  As
+well as having a new, shiny web UI, it relies entirely on GitHub Oauth for
```

Comment:
```suggestion
well as having a web UI, it relies entirely on GitHub OAuth for
```

Bear in mind that everything you write will likely stay in this document with minimal edits forevermore. Better to be less fun and more formal :)

Reply:
hey, we can't test the elections app until this PR merges.  Can we save cosmetic edits for iterations?